packages/nextjs/app/page.tsx

```diff
@@ -1,52 +1,247 @@
 "use client";
 
+import { useEffect, useState } from "react";
+import Image from "next/image";
 import Link from "next/link";
 import type { NextPage } from "next";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
 
-const Home: NextPage = () => {
+const images: string[] = []; // TODO: add the images of batch members
```

Comment:
Sorry, I don't think we'll have time to add more features to the project, so it's better to remove this comment.